mirage-clock-unix.opam

```diff
@@ -3,18 +3,22 @@ authors: "The MirageOS team"
 maintainer: "anil@recoil.org"
 homepage: "https://github.com/mirage/mirage-clock"
 bug-reports: "https://github.com/mirage/mirage-clock/issues"
+license: "ISC"
 tags: [
   "org:mirage"
   "org:xapi-project"
 ]
-build: [make "unix-build"]
-install: [make "unix-install"]
-remove: ["ocamlfind" "remove" "mirage-clock-unix"]
 depends: [
-  "ocamlfind"
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
   "mirage-types" {>= "0.3.0"}
   "mirage-types-lwt"
   "lwt"
-  "cstruct" {>= "2.0.0"}
+]
+build: [ "ocaml" "pkg/pkg.ml" "build" "--pkg-name" name "--pinned" pinned ]
```

Comment:
If this has to go through the current Ocaml OPAM repository, CI will scream on `pinned` replace with `"%{pinned}%"`